docker-compose.yml

```diff
@@ -2,15 +2,15 @@ version: "3.8"
 
 services:
   api:
-    image: "neurobagel/api:${API_TAG:-latest}"
+    image: "neurobagel/api:${NB_API_TAG:-latest}"
     ports:
       - "8000:8000"
     environment:
-      USERNAME: ${USERNAME}
-      PASSWORD: ${PASSWORD}
-      GRAPH_ADDRESS: ${GRAPH_ADDRESS:-graph}
-      GRAPH_DB: ${GRAPH_DB}
-      RETURN_AGG: ${RETURN_AGG}
+      NB_GRAPH_USERNAME: ${NB_GRAPH_USERNAME}
```

Comment:
just for my understanding: is this the place where we can independently decide what the env variables should be called outside docker (i.e. on user machine) and inside docker (i.e. our app)? 

In other words, Could I change this line to
```
HEHE_YOU_WILL_NEVER_KNOW_VAR: ${NB_GRAPH_USERNAME}
```
and be OK as long as that's what I am using internally in my `os.getenv` calls?

Reply:
Yes. In your example `NB_GRAPH_USERNAME` would need to be set in the host env, it gets assigned to the variable `HEHE_YOU_WILL_NEVER_KNOW_VAR`  inside Docker (in the container's env), and we would update the corresponding global variable in utility.py to be
```python
GRAPH_USERNAME = EnvVar(
    "HEHE_YOU_WILL_NEVER_KNOW_VAR", os.environ.get("HEHE_YOU_WILL_NEVER_KNOW_VAR")
)
```